core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/ReflectiveClassBuildItem.java

```diff
@@ -144,9 +146,17 @@ public static ReflectiveClassBuildItem serializationClass(String... classNames)
         return ReflectiveClassBuildItem.builder(classNames).serialization().build();
     }
 
+    @Deprecated(since = "3.14", forRemoval = true)
     ReflectiveClassBuildItem(boolean constructors, boolean queryConstructors, boolean methods, boolean queryMethods,
             boolean fields, boolean weak, boolean serialization,
             boolean unsafeAllocated, String... className) {
+        this(constructors, queryConstructors, methods, queryMethods, fields, false, weak, serialization, unsafeAllocated,
+                className);
+    }
```

Comment:
If it's not used, you actually don't need that given it's not public.

Reply:
It's only used by two the already deprecated public constructors, but I added the Deprecation notice to make sure people will stop using it internally as well.